.github/workflows/main.yml

```diff
@@ -9,418 +9,388 @@ on:
-  #         key: ${{ env.CACHE_KEY_ANDROID }}
-  #     - name: If no cached image found
-  #       if: steps.load-cached-android-image.outputs.cache-hit != 'true'
-  #       run: echo "ERROR == Expected to find image from cache $CACHE_KEY_ANDROID"; exit -1
-  #     - name: Load android image from cache
-  #       run: docker load < /tmp/imgcache/img_android.rar
-
-  #     - name: Test android wrapper
-  #       run: |
-  #         # docker run --name test-android-wrapper -v $PWD:/home/indy/libvcx-absa:rw $DOCKER_IMG_NAME_ANDROID \
-  #         docker run --name test-android-wrapper $DOCKER_IMG_NAME_ANDROID \
-  #                             bash -c '(cd $HOME/libvcx-absa/libvcx && ./android.wrapper.test.sh x86)'
-
-  publish-android-wrapper-device:
-    runs-on: ubuntu-16.04
-    needs: [workflow-setup, build-image-android]
-    env:
-      DOCKER_BUILDKIT: 1
-    steps:
-      - name: Load up custom variables
-        run: |
-          echo ::set-env name=CACHE_KEY_ANDROID::$(echo ${{needs.workflow-setup.outputs.CACHE_KEY_ANDROID}})
-          echo ::set-env name=DOCKER_IMG_NAME_ANDROID::$(echo ${{needs.workflow-setup.outputs.DOCKER_IMG_NAME_ANDROID}})
-      - name: Git checkout
-        uses: actions/checkout@v2
-
-      - name: Load android image cache
-        id: load-cached-android-image
-        uses: actions/cache@v2
-        with:
-          path: /tmp/imgcache
-          key: ${{ env.CACHE_KEY_ANDROID }}
-      - name: If no cached image found
-        if: steps.load-cached-android-image.outputs.cache-hit != 'true'
-        run: echo "ERROR == Expected to find image from cache $CACHE_KEY_ANDROID"; exit -1
-      - name: Load android image from cache
-        run: docker load < /tmp/imgcache/img_android.rar
-
-      - name: Build, run android wrapper tests, and publish artifacts
-        run: |
-          # TODO: Do we need all those architectures for device?
-          # docker run --name test-android-wrapper -v $PWD:/home/indy/libvcx-absa:rw $DOCKER_IMG_NAME_ANDROID \
-          docker run --name publish-android-wrapper $DOCKER_IMG_NAME_ANDROID \
-                              bash -c '(cd $HOME/libvcx-absa && ./wrappers/java/ci/android.build.sh arm armv7 arm64)'
-          docker_id=$(docker ps -a | grep publish-android-wrapper | grep Exited | tail -n 1 | cut -d ' ' -f 1)
-          docker_image_id=$(docker images | grep $DOCKER_IMG_NAME_ANDROID | perl -pe 's/\s+/ /g' | cut -d ' ' -f 3)
-          mkdir -p /tmp/artifacts/aar
-          docker cp ${docker_id}:/home/indy/artifacts/aar /tmp/artifacts/aar
-          docker rm ${docker_id} > /dev/null
-          docker rmi ${docker_image_id} > /dev/null
-
+            ./wrappers/ios/ci/build.sh
       - uses: actions/upload-artifact@v2
         with:
-          name: libvcx-android-device
-          path: /tmp/artifacts/aar
-
-#   publish-android-wrapper-universal:
+          name: libvcx-ios-universal
+          path: /tmp/artifacts/vcx
+
+#   workflow-setup:
```

Comment:
I suppose you forgot to remove comments in GA?

Reply:
They are commented out to speed up development.